ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala

```diff
@@ -172,8 +172,8 @@ class IntegrityChecker[LogResult](commitStrategySupport: CommitStrategySupport[L
       case Success((startTime, _)) =>
         Success {
           println("Successfully indexed all updates.".green)
-          val durationSeconds = Duration.fromNanos(System.nanoTime() - startTime).toSeconds
-          val updatesPerSecond = readService.updateCount() / durationSeconds.toDouble
+          val durationSeconds = Duration.fromNanos(System.nanoTime() - startTime).toMillis / 1000.0d
```

Comment:
I'd prefer an explicit `toDouble` here.

Also, the `d` suffix isn't necessary; doubles are the default.

```suggestion
          val durationSeconds = Duration.fromNanos(System.nanoTime() - startTime).toMillis.toDouble / 1000.0
```

Reply:
You need to reformat.